contrib/largeNbDicts/largeNbDicts.c

```diff
@@ -703,6 +706,23 @@ static int benchMem(slice_collection_t dstBlocks,
     }
     DISPLAY("\n");
 
+    char* csvFileName = malloc(strlen(exeName) + 5);
+    strcpy(csvFileName, exeName);
+    strcat(csvFileName, ".csv");
+    FILE* csvFile = fopen(csvFileName, "r");
+    if (!csvFile) {
+        csvFile = fopen(csvFileName, "wt");
+        assert(csvFile);
+        fprintf(csvFile, "%s\n", exeName);
+    } else {
+        free(csvFileName);
+        csvFile = fopen(csvFileName, "at");
```

Comment:
You've freed the wrong thing
```suggestion
        fclose(csvFile);
        csvFile = fopen(csvFileName, "at");
```

Reply:
ah whoops thanks